dashboard/test/models/script_test.rb

```diff
@@ -55,29 +55,29 @@ class ScriptTest < ActiveSupport::TestCase
 
   setup do
```

Comment:
Should this file be renamed to `unit_test.rb`? And the class renamed to `UnitTest`?

Reply:
ooh, yes, good catch!

Reply:
the ambiguity of "unit test" is perhaps the most unfortunate part of this rename 🤷‍♂️